### DISCUSSION TITLE
Offer to help with some stuff

### DISCUSSION THREAD

**Discussion by VinylNerd** [MultimediaLucario/Lenovo-ThinkPad-T480#7] [General]
Hey, I have been doing some work on T480 myself, have 3 monitors up on ventura with full working airdrop and continuity including working wireless continuity camera, have thunderbolt and 10gbps usb working. have NVMe main drive in 2.5'' slot

not wanting to go to sonoma because broadcom wifi required OCLP and id rather not loose SIP for Sonoma, no features i really want or need

shoot me a message if you want to chat about stuff on whatsapp or signal

**Comment by MultimediaLucario** [comment]
Just curious, did you ever manage to get AirDrop on Intel Wi-Fi working?

**Reply by VinylNerd** [reply]
Yes i have working airdrop and continuity, with continuity camera working
